forms/scraper/keyword_result_form.go

```diff
@@ -0,0 +1,98 @@
+package forms
+
+import (
+	"fmt"
+	"net/url"
+
+	. "go-crawler-challenge/forms"
+	"go-crawler-challenge/models"
+	"go-crawler-challenge/transactions"
+
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/beego/beego/v2/core/validation"
+)
+
+type KeywordResultForm struct {
+	Keyword  string `valid:"Required; MaxSize(128)"`
+	VisitUrl string `valid:"Required; MaxSize(128)"`
```

Comment:
Not sure about the naming here (and also in the database) 🤔 `searchUrl` or simply `url` would be better alternatives.

Reply:
https://github.com/Lahphim/go-crawler-challenge/pull/55/commits/89a9296eefd8bc3a9d625ce8d60c688b17f7d198